pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1336,12 +1337,17 @@ impl RemoteTimelineClient {
                     // In-place replace of Initialized to Stopped can be done with the help of https://github.com/Sgeo/take_mut
                     // but for this use case it doesnt really makes sense to bring unsafe code only for this usage point.
                     // Deletion is not really perf sensitive so there shouldnt be any problems with cloning a fraction of it.
+                    let (visible_remote_consistent_lsn_tx, visible_remote_consistent_lsn_rx) =
+                        tokio::sync::mpsc::channel(16);
```

Comment:
I wonder what is this for?

Reply:
Unresolving because this will lose any messages there were in the queue.

Reply:
This has gone away in db9a49ed918630be83f6b8dd0512e56cecb3be8a